daprdocs/content/en/reference/api/jobs_api.md

```diff
@@ -63,6 +64,39 @@ Entry                  | Description                                | Equivalent
 @daily (or @midnight)  | Run once a day, midnight                   | 0 0 0 * * *
 @hourly                | Run once an hour, beginning of hour        | 0 0 * * * *
 
+#### failure_policy
+
+`failure_policy` specifies how the job should handle failures.
+
+It can be set to `constant` or `drop`.
+- The `constant` policy retries the job based on the configuration.
+  - `max_retries` configures how many times the job should be retried. Not setting this makes it retry indefinitely.
```

Comment:
```suggestion
  - `max_retries` configures how many times the job should be retried. Defaults to retrying indefinitely.
```

Reply:
Not sure if you can set -1 here or 0? If so them would add something like:

"-1 denotes an unlimited number of retries, while 0 means the request will not be retried."